.eslintrc

```diff
@@ -4,5 +4,8 @@
     "@readme/eslint-config/react"
   ],
   "parser": "babel-eslint",
-  "root": true
+  "root": true,
+  "rules": {
+    "no-cond-assign": ["error", "except-parens"]
```

Comment:
@rafegoldberg does this seem fair?

Reply:
Shouldn't `"except-parens"` already be the default for this rule? Maybe we're overriding it in our core [ESLint config](https://github.com/readmeio/eslint-config)? (In which case we should update the rule in that repo too!) But yeah, I'm okay with this.